File: Programming Fundamentals/Arrays - Exercises/Index of Letters/IndexOfLetters.cs
namespace Index_of_Letters
{
    using System;

    public class IndexOfLetters
    {
        public static void Main(string[] args)
        {
            char[] letters = new char[26];
            for (int i = 97; i < letters.Length + 97; i++)
            {
                letters[i - 97] = (char)i;
            }

            string input = Console.ReadLine();
            for (int i = 0; i < input.Length; i++)
            {
                for (int j = 0; j < letters.Length; j++)
                {
                    if (letters[j] == input[i])
                    {
                        Console.WriteLine($"{input[i]} -> {j}");
                    }
                }
            }
        }
    }
}

File: Software Technologies/Java/IndexOfLetters.java
public class IndexOfLetters {
    public static void main(String[] args) {
        java.util.Scanner scanner = new java.util.Scanner(System.in);

        String input = scanner.next().toLowerCase();

        for (int i = 0; i < input.length(); i++) {
            char currentChar = input.charAt(i);
            System.out.printf("%s -> %d\n", currentChar, (int)(currentChar - 'a'));
        }
    }
}
